priv/repo/migrations/20170607142123_create_setting.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateSetting do
+  use Ecto.Migration
+
+  def change do
+    create table(:settings) do
```

Comment:
I was thinking about this table...

What do you @ghatighorias @rhnonose  think about having the settings table with the columns name and value and each setting is a different row? Somewhat like a key-value store